src/index.js

```diff
@@ -224,7 +244,7 @@ const createListItem = (location, status, lng, lat) => {
       popup.remove()
     } else {
       closePopups()
-      toggleSidePane()
+      toggleSidePane($locationsButton)
```

Comment:
I'm pretty sure someone refactored the `toggleSidePane` method in the last day or so, and I'm not sure if this should be `toggleSidePane()` or `toggleSidePane($locationsButton)`

Reply:
Ach, you're right. Fixed.